timescale-container/test/crash/export-lusovini-data.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!